tests/acceptance/features/apiSharingNg1/sharedWithMe.feature

```diff
@@ -5232,3 +5232,35 @@ Feature: an user gets the resources shared to them
     And the json response should contain the following shares:
       | textfile.txt  |
       | FolderToShare |
+
+  @env-config
+  Scenario: share a folder with denied permission role (Personal Space)
+    Given using spaces DAV path
+    And the administrator has enabled the permissions role "Denied"
+    And user "Alice" has created folder "FolderToShare"
+    And user "Alice" has sent the following resource share invitation:
+      | resource        | FolderToShare |
+      | space           | Personal      |
+      | sharee          | Brian         |
+      | shareType       | user          |
+      | permissionsRole | Denied        |
+    When user "Brian" lists the shares shared with him using the Graph API
+    Then the HTTP status code should be "200"
+    And user "Brian" should not have a share "FolderToShare" shared by user "Alice" from space "Personal"
```

Comment:
add a file inside `FolderToShare` and check that sharee cannot download it
```suggestion
    And user "Brian" should not have a share "FolderToShare" shared by user "Alice" from space "Personal"
    And user "Brian" should not be able to download file "FolderToShare/lorem.txt" from space "Shares"
```

Reply:
Done :+1: